src/bootstrap/builder.rs

```diff
@@ -1365,14 +1365,17 @@ impl<'a> Builder<'a> {
             },
         );
 
-        if target.contains("linux") || target.contains("windows") {
-            rustflags.arg("-Zunstable-options");
+        // FIXME(davidtwco): allow this on Windows once #95612 is in the bootstrap compiler
```

Comment:
You should be able to gate this on the compiler stage, right? That way we still would pass the option to freshly built compilers which should do the right thing.

Also please leave a cfg(bootstrap) in the comment so that we can find it when searching for what needs bumping after the bootstrap compiler update.

Reply:
Fixed